core/grpc/src/rpc_types.rs

```diff
@@ -247,9 +248,9 @@ pub enum PubDataType {
     VerfKey,     // Type for the servers public verification keys
     VerfAddress, // The ethereum address of the KMS core, needed for KMS signature verification
     DecompressionKey,
-    CACert,                // Certificate that signs TLS certificates used by MPC nodes
-    CustodianSetupMessage, // Backup custodian public keys (self-signed)
-    PublicEncKey,          // Classical non-FHE Public encryption key, e.g. used for backup
+    CACert, // Certificate that signs TLS certificates used by MPC nodes // TODO I think this should be in private since we don't trust the public to never be modified
```

Comment:
I did not date modify it since @mkmks did this part.
But it seems to me with the current security model it should be moved.